src/general/Mutations.cs

```diff
@@ -350,19 +352,23 @@ private OrganelleDefinition GetRandomOrganelle(bool isBacteria)
 
                 for (int side = 1; side <= 6; ++side)
                 {
-                    // Offset by hex offset
-                    result.Position = pos + Hex.HexNeighbourOffset[(Hex.HexSide)side];
-
-                    // TODO: checking one or two extra hexes in the direction would make this succeed more often
-
-                    // Check every possible rotation value.
-                    for (int rotation = 0; rotation <= 5; ++rotation)
+                    for (int multiplier = 1; multiplier <= 3; ++multiplier)
```

Comment:
I think `multiplier` should be named `radius` here to reflect better with other similar code already in the game.